pkgs/by-name/fa/fahrplan/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "fahrplan";
+  version = "1.1.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "dbrgn";
+    repo = "fahrplan";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-2QVaA2zqqFuxjkKbDXo+SNzkrDCxXiSYMUOftQ++wO4=";
+  };
+
+  build-system = with python3.pkgs; [
+    setuptools
+  ];
+
+  dependencies = with python3.pkgs; [
+    dateutil
```

```suggestion
    python-dateutil
```
`dateutil` is an alias